packages/react/src/components/FluidNumberInput/index.tsx

```diff
@@ -4,7 +4,9 @@
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
  */
-
+import { FluidNumberInputProps } from './FluidNumberInput';
+import { FluidNumberInputSkeletonProps } from './FluidNumberInput.Skeleton';
+export { type FluidNumberInputProps, type FluidNumberInputSkeletonProps };
```

Comment:
@riddhybansal in addition to the `export`, the `type` identifier needs to be on the `import` statements as well. 

I think this should resolve the "`The exported identifier "FluidNumberInputProps" is not declared in Babel's scope tracker as a JavaScript value binding, ...`" errors